### DISCUSSION TITLE
HELP

### DISCUSSION THREAD

**Discussion by jun547260** [ExpressLRS/ExpressLRS#1766] [Q&A]
I keep getting this.
I'm using the "Jumper t-lite v2 Elrs" , and I'm trying to connect to the "Mobula 7 1s Elrs"
I'm trying to download the 2.x.x maintenance version but it keeps failing.
Please help me.

![20220812_095152](https://user-images.githubusercontent.com/111082299/184265921-4b01cb07-9802-496c-be5f-bbcb98694df8.png)

**Comment by SunjunKim** [comment]
https://www.youtube.com/watch?v=9jyAVHi6lrI
